piety-promoted-v2/modernized/12-chapter-11.adoc

```diff
@@ -836,25 +836,25 @@ and the goodness and love of God came over all present.
 The young man making melody,
 seemed to have an evidence of that divine favor which
 his soul had been so deeply engaged for;
-and soon after said, O Lord, what shall my soul say?
+and soon after said, '`Lord, what shall my soul say?
```

Comment:
I think you're right to add the quote here, but I believe the `O` should be retained as well.